tasks-by-users/Rehan-Khan-Engineer/Bikes-webpage/Rehan.css

```diff
@@ -0,0 +1,42 @@
+/* Bike ki Dukaan */
+body{ margin: 0; font-family: Arial; background: white;}
```

Comment:
use hash code instead @Rehan-Khan-Engineer

Reply:
i think its fine to use color name @mmusk